connectdb.js

```
@@ -3,25 +3,26 @@ let mysql = require('async-mysql')
 let connection = null;
 
 //turn MySQL connect into async function that can be called multiple times
-async function connectDB(){
+const connectDB = () => {
     
     if(connection != null) {
         //already connected
         return connection
     }
 
-    console.log("connecting to MySQL")
-    return await new Promise((res,rej)=>{
-        mysql.connect({
+    console.log("connecting to MySQL");
+    try {
+        connection = mysql.connect({
             user     : 'dev',
             password : 'animal',
             database : 'world'
-        }).then((con)=>{
-            connection = con;
-            console.log("connected")
-            res(connection)
         })
-    })
+    } catch (err) {
+        return null
```

Probably should let connectDB error out instead of returning null